Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -552,6 +554,7 @@ extension ActivityStreamPanel {
                 site.setBookmarked(true)
             })
         }
+        self.telemetry.reportEvent(.Bookmark, source: pingSource, position: index)
```

Comment:
This should be inside the scope of the bookmarkAction block instead of function contextMenuForSite or else this will get called every time the context menu is opened.